crypto/libtomcrypt/Makefile

```diff
@@ -476,10 +476,13 @@ $(LIBTOMCRYPT_UNPACKNAME): $(LIBTOMCRYPT_ZIP)
 	$(Q) patch -p0 < ltcrypt-flush.patch # Missing stdout flush in demo app
 	$(Q) touch $(LIBTOMCRYPT_UNPACKNAME)
 
+# Download and unpack tarball if no git repo found
+ifeq ($(wildcard $(LIBTOMCRYPT_UNPACKNAME)/.git),)
```

Comment:
should we guard line 466-477 too?

Reply:
I think no need for that ? Small guard range is clear to understand and these dangling targets have no side effects for system.

Reply:
Ok.